adagrams/game.py

```diff
@@ -1,11 +1,128 @@
+import random
+
 def draw_letters():
-    pass
+    LETTER_POOL = {
+        "A": 9,
+        "B": 2,
+        "C": 2,
+        "D": 4,
+        "E": 12,
+        "F": 2,
+        "G": 3,
+        "H": 2,
+        "I": 9,
+        "J": 1,
+        "K": 1,
+        "L": 4,
+        "M": 2,
+        "N": 6,
+        "O": 8,
+        "P": 2,
+        "Q": 1,
+        "R": 6,
+        "S": 4,
+        "T": 6,
+        "U": 4,
+        "V": 2,
+        "W": 2,
+        "X": 1,
+        "Y": 2,
+        "Z": 1
+    }
+
+    hand_letters = []
+    letters = list(LETTER_POOL.keys())
+    
+    while len(hand_letters) < 10:
```

Comment:
Nicely done! This will respect the letter quantity limits, which is great! One thing to note, in terms of randomly selecting letters, as your code is structured, there's always a 1/26 chance of selecting a letter, whereas the letter values should represent the distribution of letters up for random selection; in other words, there should be a 9/98 chance of drawing an A, a 2/98 chance of drawing a B, and so on. It's not something we have tests coverage on, but a good thing to consider!